### PR TITLE
volta: new cask

### PR DESCRIPTION
**Pull request by danielbayley**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/volta-net.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  url "https://volta-build.s3.fr-par.scw.cloud/Volta-#{version}.dmg",
+      verified: "volta-build.s3.fr-par.scw.cloud/"
+  name "Volta"
+  desc "GitHub issues and notifications"
+  homepage "https://volta.net/"
+
+  livecheck do
+    url "https://volta.net/changelog"
+    strategy :page_match
```
Comment (bevanjkay) [inline Casks/volta-net.rb:13]: This line can be omitted as it is the default strategy.

File: Casks/volta-net.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+cask "volta-net" do
```
Comment (bevanjkay) [inline Casks/volta-net.rb:1]: We do have some casks that share a token with a formula

**Review by bevanjkay — DISMISSED** [review]

File: Casks/volta-net.rb (hunk 2 above)
Comment (danielbayley) [reply Casks/volta-net.rb:1]: > We do have some casks that share a token with a formula

So how to avoid the error?

Comment (bevanjkay) [reply Casks/volta-net.rb:1]: It can be skipped on maintainer review, and is only checked on initial submission.

**Review by miccal — DISMISSED** [review]

**Review by miccal — APPROVED** [review]

**Comment by razvanazamfirei** [comment]
I feel like cask/formula token conflicts should be skippable. Alternatively, we could rename this to `nuxtlabs-volta`.

**Review by p-linnane — APPROVED** [review]

**Comment by bevanjkay** [comment]
I am happy to merge as-is

**Pull request merged**